src/CartItem.java

```diff
@@ -22,11 +22,21 @@ public int getQuantity() {
     }
 
     public void setQuantity(int quantity) {
+        if (quantity < 0) {
+            throw new IllegalArgumentException("Quantity cannot be negative.");
+        }
```

Comment:
I think we should move the validation of the input into a separate method maybe called `isQuantityValid()`to be more consistent with what  @simonlartey did in `Book.java`.
We should also restrict the quantity from being 0 too in addition to it not being negative.
We don't want the situation where an item is in the Cart but it's quantity is 0; In that case, the item should rather be removed from the Cart.
```suggestion
       if (!isQuantityValid(quantity)){
           throw new IllegalArgumentException("Quantity cannot be negative.")
       }
```